modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -333,6 +333,12 @@ class Form extends Component {
 		return ruleBuilderVisible && this.isFormBuilderView();
 	}
 
+	isShowEntries() {
+		const {entriesVisible} = this.state;
```

Comment:
rename to showEntries

Reply:
it's probably better to create a feature flag to show/hide the entries tab, we will hide it until Reports is completely implemented.